api/v1alpha1/hostedcluster_types.go

```diff
@@ -1834,10 +1838,8 @@ type ClusterConfiguration struct {
 // +kubebuilder:storageversion
 // +kubebuilder:subresource:status
 // +kubebuilder:printcolumn:name="Version",type="string",JSONPath=".status.version.history[?(@.state==\"Completed\")].version",description="Version"
-// +kubebuilder:printcolumn:name="KubeConfig",type="string",JSONPath=".status.kubeconfig.name",description="KubeConfig Secret"
```

Comment:
These should probably have been a in separate PR but removing the `KubeConfig` and `Reason` columns since those will be redone as messages on the Progressing